src/routers/course.ts

```diff
@@ -36,9 +36,8 @@ courseRouter.post('/', async (req, res) => {
 				owner: userID,
 				name: req.body.name,
 			});
-			await create(course);
-			console.log('Post Course');
-			res.send('Post courseRouter: ' + req.body.name);
+			const created = await create(course);
+			res.json(created);
```

Comment:
Are there any objections to using `res.json` here instead of calling `created.toJSON()`?

Reply:
This is fine. I looked up the documentation for res.json and it called `JSON.stringify` internally, and `JSON.stringify` will try to call the `toJSON` method on the object if it exists, which it does.